cg_hermes/config/mutant.py

```diff
@@ -1,93 +1,100 @@
 MUTANT_COMMON_TAGS = {
+
     frozenset({"sampleinfo", "runinfo"}): {
         "is_mandatory": True,
         "bundle_id": True,
         "tags": ["config"],
         "used_by": ["storage", "audit"],
     },
-    frozenset({"variants", "genotyping"}): {
-        "is_mandatory": False,
-        "tags": ["vcf", "vcf-report"],
-        "used_by": ["deliver"],
+    frozenset({"runtime-settings", "runinfo"}): {
+        "is_mandatory": True,
+        "tags": ["mutant-config"],
+        "used_by": ["storage", "audit"],
     },
-    frozenset({"variants", "variant-calling"}): {
-        "is_mandatory": False,
-        "tags": ["tsv", "vcf-report"],
-        "used_by": ["deliver"],
+    frozenset({"concatination", "forward-reads"}): {
+        "is_mandatory": True,
+        "tags": ["fastq", "forward-strand"],
+        "used_by": ["storage"],
     },
-    frozenset({"SARS-CoV-2-qc", "result_aggregation"}): {
-        "is_mandatory": False,
-        "tags": ["qc-report", "csv", "visualization"],
-        "used_by": ["deliver"],
+    frozenset({"reverse-reads", "concatination"}): {
+        "is_mandatory": True,
+        "tags": ["fastq", "reverse-strand"],
+        "used_by": ["storage"],
     },
-    frozenset({"SARS-CoV-2-type", "typing"}): {
-        "is_mandatory": False,
-        "tags": ["typing-report", "visualization"],
-        "used_by": ["deliver"],
+    frozenset({"consensus"}): {
+        "is_mandatory": True,
+        "tags": ["ks", "fastq", "consensus"],
+        "used_by": ["deliver", "storage"],
     },
-    frozenset({"SARS-CoV-2-json", "result_aggregation"}): {
-        "is_mandatory": False,
-        "tags": ["typing-report", "visualization"],
+    frozenset({"runinfo", "logfile"}): {
+        "is_mandatory": True,
+        "tags": ["mutant-log"],
+        "used_by": ["audit"],
+    },
+    frozenset({"multiqc-json"}): {
+        "is_mandatory": True,
+        "tags": ["multiqc-json"],
+        "used_by": ["vogue"],
+    },
+    frozenset({"multiqc-html"}): {
+        "is_mandatory": True,
+        "tags": ["ks", "multiqc-html"],
         "used_by": ["deliver"],
     },
-    frozenset({"SARS-CoV-2-sum", "report"}): {
-        "is_mandatory": False,
-        "tags": ["summary", "csv", "visualization"],
+    frozenset({"ks-results"}): {
+        "is_mandatory": True,
+        "tags": ["ks", "pangolin", "typing-report", "visualization", "csv"],
         "used_by": ["deliver"],
     },
-    frozenset({"SARS-CoV-2-var", "report"}): {
-        "is_mandatory": False,
-        "tags": ["variants", "csv", "visualization"],
+    frozenset({"ks-aux-results"}): {
+        "is_mandatory": True,
+        "tags": ["ks", "pangolin", "typing-report", "visualization", "csv"],
```

Comment:
can we make the ks-results and ks-aux-results have unique and distinctive tags as well? Common "pangolin" is good for delivery though

Reply:
Gave them unique names. Also created the group "ks-delivery"